.travis.yml

```diff
@@ -14,7 +14,9 @@ before_script:
   - travis_retry composer self-update
   - travis_retry composer update --no-interaction --prefer-source
 
-script: make test-coverage-clover
+script:
```

Comment:
Thats what I did with array-filter. The environment thing could work. Something like:

```
env:
  - PHP_VERSION=5.6
  - PHP_VERSION=7
  - PHP_VERSION=hhvm

matrix:
  allow_failures:
    - env: PHP_VERSION=7
    - env: PHP_VERSION=hhvm
```

**Dockerfile:**
```
use php:${PHP_VERSION:-5.6}
```

Then each command is the same

Be nice to use the php images, but